### ISSUE TITLE
Parallel Poisson example

### ISSUE THREAD

**Issue by saidctb** [gismo/gismo#475]
Dear All,
It seems that the file `examples/poisson2_example.cpp` runs sequentially , do you have its equivalent in parallel mode?

thanks for your help

**Comment by mmoelle1** [comment]
The application `examples/poisson2_example.cpp` makes use of the expression assembler which is still sequential in the version of the `stable` branch. We are working on an Open-MP parallelized expression assembler that will soon be merged into the `stable` branch.

**Comment by filiatra** [comment]
Hi there,
the parallel version (openMP) is now merged in stable in the latest commits.
Let us know how it works for you!
